ggj25/scripts/game_mode_selection.gd

```diff
@@ -10,6 +10,7 @@ extends Control
 
 func _ready() -> void:
 	music.play()
+	game_1.grab_focus()
```

Comment:
When a scene is loaded we just need to make sure we give focus to a button, and if the tree is setup cleanly then godot kinda jusat does the rest.

https://docs.godotengine.org/en/stable/tutorials/ui/gui_navigation.html